scripts/solo_eight_backend.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+"""Run Solo8 back-end using multi-process robot data."""
+import argparse
+import logging
+import math
+import pathlib
+import sys
+
+import robot_interfaces
+import robot_fingers
+
+
+def main():
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        "--max-number-of-actions",
+        "-a",
+        type=int,
+        required=True,
+        help="""Maximum numbers of actions that are processed.  After this the
+            backend shuts down automatically.
+        """,
+    )
+    parser.add_argument(
+        "--first-action-timeout",
+        "-t",
+        type=float,
+        default=math.inf,
+        help="""Timeout (in seconds) for reception of first action after
+            starting the backend.  If not set, the timeout is disabled.
+        """,
+    )
+    parser.add_argument(
+        "--robot-logfile",
+        type=str,
+        help="""Path to a file to which the robot data log is written.  If not
+            specified, no log is generated.
+        """,
+    )
+    parser.add_argument(
+        "--ready-indicator",
+        type=str,
+        metavar="READY_INDICATOR_FILE",
+        help="""Path to a file that will be created once the backend is ready
+            and will be deleted again when it stops (before storing the logs).
+        """,
+    )
+    args = parser.parse_args()
+
+    log_handler = logging.StreamHandler(sys.stdout)
+    logging.basicConfig(
+        format="[Solo8_BACKEND %(levelname)s %(asctime)s] %(message)s",
+        level=logging.DEBUG,
+        handlers=[log_handler],
+    )
+
+    logging.info("Start robot backend")
+
+    # Use robot-dependent config file
+    config_file_path = "/home/robot/workspace/workspace/install/robot_fingers/share/robot_fingers/config/soloeight.yml"
```

Comment:
Instead of hard coding the path, you can use `get_package_share_directory` from `ament_index_python` to dynamically get the path to the config files.  Then it will work independent on the location of the workspace.  See [here](https://github.com/open-dynamic-robot-initiative/robot_fingers/blob/b07f1790181259de4e7bb4ba4d74aaff3a866219/demos/demo_single_finger_position_control.py#L21) for an example.